src/views/TopPage.vue

```diff
@@ -1,4 +1,15 @@
 <script setup lang="ts">
+import { ref } from "vue";
+import { GourmetStartRequest } from "../apis/generated";
+import { gourmetStartRequest } from "../apis";
+
+const x: GourmetStartRequest = {
+  station: "text",
```

Comment:
ここが"text"という文字列になっているので, 二重引用符を剥してください 
```suggestion
  station: text,
```

Reply:
refを使ってるので, `text.value`が正しかったです
```suggestion
  station: text.value,
```

Reply:
嘘でした